lib/decanter/parser/boolean_parser.rb

```diff
@@ -5,9 +5,23 @@ class BooleanParser < ValueParser
       allow TrueClass, FalseClass
 
       parser do |val, options|
-        raise Decanter::ParseError.new 'Expects a single value' if val.is_a? Array
-        next if (val.nil? || val === '')
-        [1, '1'].include?(val) || !!/true/i.match(val.to_s)
+        normalized_val = normalize(val)
+        next if normalized_val.nil?
+
+        true_values = ['1', 'true']
+
+        option_val = options.fetch(:true_value, nil)
+        normalized_option = normalize(option_val)
```

Comment:
Out of curiosity, why are you only allowing one additional option to be specified? What if we wanted to allow "on" or "T" as well?

Reply:
the scope of the original issue seem to indicate adding the ability for one optional value. That was my interpretation of the issue, but work on a solution that will allow for additional optional values if desired.

Reply:
I also took this as a design decision (perhaps implicitly by me and adopted by Owen). I don't wholeheartedly disagree with it as this option is per input and is meant to handle the nuance of a given input. Flexibility vs intentionality, and I can see both options. Either way that implementation detail can probably also be documented for the user.

Reply:
taking the more flexible route is definitely fine, as it obviously also covers the single case